### PR TITLE
[Merged by Bors] - chore(Finset/Density): Fix statement and make lemmas simp

### PR DESCRIPTION
**Pull request by YaelDillies**
From LeanAPAP

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Finset/Density.lean [hunk 1]
```diff
@@ -110,23 +110,24 @@ lemma dens_image [Fintype β] [DecidableEq β] {f : α → β} (hf : Bijective f
     (s.image f).dens = s.dens := by
   simpa [map_eq_image, -dens_map_equiv] using dens_map_equiv (.ofBijective f hf)
 
-lemma card_mul_dens (s : Finset α) : Fintype.card α * s.dens = s.card := by
+@[simp] lemma card_mul_dens (s : Finset α) : Fintype.card α * s.dens = s.card := by
+
```
Comment (b-mehta) [inline Mathlib/Data/Finset/Density.lean:114]:
```suggestion
```

File: Mathlib/Data/Finset/Density.lean [hunk 2]
```diff
@@ -110,23 +110,24 @@ lemma dens_image [Fintype β] [DecidableEq β] {f : α → β} (hf : Bijective f
...
-lemma dens_mul_card (s : Finset α) : s.dens * Fintype.card α = s.card := by
+@[simp] lemma dens_mul_card (s : Finset α) : s.dens * Fintype.card α = s.card := by
   rw [mul_comm, card_mul_dens]
 
 section Semifield
 variable [Semifield 𝕜] [CharZero 𝕜]
 
-lemma natCast_card_mul_nnratCast_dens (s : Finset α) : (Fintype.card α * s.dens : 𝕜) = s.card :=
-  mod_cast s.card_mul_dens
+@[simp] lemma natCast_card_mul_nnratCast_dens (s : Finset α) :
```
Comment (b-mehta) [inline Mathlib/Data/Finset/Density.lean:126]: hmm I wonder if we should have automation to make cast versions of nat lemmas like this... the pattern shows up fairly often

**Review by b-mehta — APPROVED** [review]
bors d+

File: Mathlib/Data/Finset/Density.lean (hunk 2 above)
Comment (YaelDillies) [reply Mathlib/Data/Finset/Density.lean:125]: Maybe... The pattern is rather predictable

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**